lib/cc/engine/analyzers/python/parser.rb

```diff
@@ -42,10 +41,14 @@ def initialize(command, delegate)
 
           def run(input, timeout = DEFAULT_TIMEOUT)
             Timeout.timeout(timeout) do
-              child = ::POSIX::Spawn::Child.new(command, input: input, timeout: timeout)
+              IO.popen command, 'r+' do |io|
```

Comment:
Styleguide :lipstick: single quotes -> double quotes (though, they seem to be used inconsistently throughout this repo).